frontend/ui/widget/filechooser.lua

```diff
@@ -39,54 +74,65 @@ local unreadable_dir_content = {}
 
 function FileChooser:init()
     self.width = Screen:getWidth()
-    -- common dir filter
-    self.dir_filter = function(dirname)
+    -- Standard dir exclusion list
+    self.show_dir = function(dirname)
         for _, pattern in ipairs(self.exclude_dirs) do
             if dirname:match(pattern) then return false end
         end
         return true
     end
+    -- Standard file exclusion list
+    self.show_file = function(filename)
+        for _, pattern in ipairs(self.exclude_files) do
+            if filename:match(pattern) then return false end
```

Comment:
I guess it might be hard/impossible to make a single pattern from the whole list - to not have to call :match `N exclude patterns` times `N files` :/

Reply:
Yeah, those aren't REs, we don't have an OR operator ;).